fpsd/requirements/ml-requirements.in

```diff
@@ -0,0 +1,2 @@
+pandas
+tqdm
```

Comment:
Yet.... Really though, the latest version of the `pyscopg2` egg (which should be a wheel by now) has some problem that is preventing `pip_update.sh` from succeeding. The smaller our requirements file stays, the less we run into pip problems.